### DISCUSSION TITLE
No sound from my browser in Sandboxie

### DISCUSSION THREAD

**Discussion by BilleBarrett** [sandboxie-plus/Sandboxie#3835] [General]
I am running Windows 11 Insider Preview 26100.1 (ge_release) and yesterday & Sandboxie-Plus v1.13.5
 I noticed that in the sandbox I have not sound
run outside and I have souind, did I check something I should not have

**Comment by bastik-1001** [comment]
Is this limited to a specific browser or browsers in general or any app that can play sound?

**Comment by BilleBarrett** [comment]
I am sorry, for not changing this, I ended up deleting the Sandboxie.ini in the windows folder and restarted Sandboxie and reset everything and I’m ok now

 

Remote Desktop Services 
(336) 988-0891    
 ***@***.***> ***@***.***

**Comment by ghost** [comment]
Are you using Confidential Box? If you do, then add this to Sandboxie.ini for your browser sandbox "DenyHostAccess=audiodg.exe,n" . Confidential Boxes prevent many host and third party services and processes from accessing the box. If you use a third party task manager, then make sure to whitelist it like audiodg or else that third party task manager won't be able to shutdown or interact with sandboxed processes.
